### DISCUSSION TITLE
CA65 macros

### DISCUSSION THREAD

**Discussion by Adrian57** [cc65/cc65#1741] [Q&A]
Re macros in CA65. In the listing file produced with the --listing option the machine code produced by the macro is printed followed by the macro name but the mnemonics are not listed. Is there any way to get the assembler to show the expansion of the macro?

I can't find any obvious option for doing this. But as one learns to write macros, especially one with conditional expansion (.IF) etc it would be REALLY nice to see the expanded source rather than just the machine code.

**Comment by spiro-trikaliotis** [comment]
No, this is not possible (AFAIK).
I am currently working on making this available (not publically available yet), but it is more complicated than I thought, and my time is limited at the moment.
